server/models/Profile.js

```diff
@@ -0,0 +1,61 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  user_id: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+  },
+  firstname: {
+    type: String,
+  },
+  lastname: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  gender: {
+    type: String,
+  },
+  birthdate: {
+    type: Date,
+  },
+  phone_number: {
+    type: String,
+  },
+  photo_url: {
+    type: String,
+  },
+  location: {
+    city: {
+      type: String,
+      required: true,
+    },
+    province: {
+      type: String,
+      required: true,
+    },
+    country: {
+      type: String,
+      required: true,
+    },
+    index: true,
```

Comment:
I like the idea🙌🏼 Although I'm personally not 💯 if `index` works in nested paths, would be a nice exercise to find out 🤔